src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -116,88 +136,94 @@ const BaseSubBudgetCard = ({
+    }
+
+    return (
+      <Card.Section
+        title={!isRetired && (
+          <h4>
             <FormattedMessage
-              id="lcm.budgets.budget.card.available"
-              defaultMessage="Available"
-              description="Label for the available balance on the budget card"
+              id="lcm.budgets.budget.card.balance"
+              defaultMessage="Balance"
+              description="Header for the balance section of the budget card"
             />
-          </div>
-          <span className="small">
-            {isFetchingBudgets ? <Skeleton /> : formatPrice(available)}
-          </span>
-        </Col>
-        {isAssignable && (
-          <Col xs="6" md="auto" className="mb-3 mb-md-0">
-            <div className="small font-weight-bold">
+          </h4>
+        )}
+        muted
+      >
+        <Col className="d-flex justify-content-start w-md-75">
+          {!isRetired && (
+            <>
+              <Col xs="6" md="auto" className="mb-3 mb-md-0 ml-n4.5">
+                <div className="small font-weight-bold">
+                  <FormattedMessage
+                    id="lcm.budgets.budget.card.available"
+                    defaultMessage="Available"
+                    description="Label for the available balance on the budget card"
+                  />
+                </div>
+                <span className="small">
+                  {isFetchingBudgets ? <Skeleton /> : formatPrice(available)}
+                </span>
+              </Col>
+              {isAssignable && (
+                <Col xs="6" md="auto" className="mb-3 mb-md-0">
+                  <div className="small font-weight-bold">
+                    <FormattedMessage
+                      id="lcm.budgets.budget.card.assigned"
+                      defaultMessage="Assigned"
+                      description="Label for the assigned balance on the budget card"
+                    />
+                  </div>
+                  <span className="small">
+                    {isFetchingBudgets ? <Skeleton /> : formatPrice(pending)}
+                  </span>
+                </Col>
+              )}
+            </>
+          )}
+          <Col xs="6" md="auto" className={classNames('mb-3 mb-md-0', { 'ml-n4.5': isRetired })}>
+            <div className={classNames('font-weight-bold', { h4: isRetired, small: !isRetired })}>
               <FormattedMessage
-                id="lcm.budgets.budget.card.assigned"
-                defaultMessage="Assigned"
-                description="Label for the assigned balance on the budget card"
+                id="lcm.budgets.budget.card.spent"
+                defaultMessage="Spent"
+                description="Label for the spent balance on the budget card"
               />
             </div>
-            <span className="small">
-              {isFetchingBudgets ? <Skeleton /> : formatPrice(pending)}
+            <span className={classNames({ 'font-size-base': !isRetired })}>
```

Comment:
I believe this should be `isRetired`, not `!isRetired`. As is, there are now font-size discrepancies between the dollar amounts for "Available", "Assigned", and "Spent" on non-retired budgets. The retired budgets are still rendering 14px font size (not 18px).

<img width="452" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/4c43be85-758b-4998-aa46-1b71700fb4a2">

<img width="472" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/e70e8864-57e6-4838-bb65-b84a8630a4e7">

I would also recommend either keeping the existing `small` class name for the time being, even if it's not actually having an effect OR otherwise remove _all_ instances of the `small` class name throughout, e.g. it's still used here:

```jsx
<div className={classNames('font-weight-bold', { h4: isRetired, small: !isRetired })}>
```

---

It should look like this:

### Non-retired budgets

<img width="452" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/9518c74c-b3e8-46c0-93d1-1679a1a0525a">

### Retired budgets

<img width="451" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/fbff77f3-a81b-4b0f-902c-6911760c1db6">